uwp/DataGrid/Unbound-Column.md

```diff
@@ -365,7 +365,7 @@ In the below code snippet, `Foreground` and `Background` of the cells in GridUnB
 
 ![Unbound-Column_img5](Unbound-Column_images/Unbound-Column_img5.png)
 
-You can refer the [Styling](http://help.syncfusion.com/uwp/sfdatagrid/column-types#styling-gridcolumn) section of `GridColumn` for more information.
+You can refer the [Styling](https://help.syncfusion.com/uwp/datagrid/getting-started#styling-gridcolumn) section of `GridColumn` for more information.
```

Comment:
Use this link - https://help.syncfusion.com/uwp/datagrid/column-types#styling-gridcolumn

Reply:
done